mathics/eval/numbers.py

```diff
@@ -131,6 +138,13 @@ def cancel(expr):
             return expr
 
 
+def is_number(expr) -> bool:
```

Comment:
`is_real_number` maybe?

Reply:
I have changed this to `is_integer_rational_or_real()` which is unambiguous.